src/pytest_docker/plugin.py

```diff
@@ -29,6 +29,25 @@ def execute(command, success_codes=(0,)):
     return output
 
 
+def pytest_addoption(parser):
```

Comment:
I think you need to import this somewhere otherwise it's not "parsed" and seen by `pytest`, because for me it's not working otherwise